target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -158,6 +158,7 @@ dir-615-h1)
 	set_wifi_led "rt2800pci-phy0::radio"
 	;;
 dir-620-d1|\
+dwr-921-c1|\
```

Comment:
`dlink,dwr-921-c1`  ?

Reply:
I prefer to be coherent with the format used in the file.